tools/function_finder/function_finder_psx.py

```diff
@@ -176,8 +146,8 @@ def find_wip(o):
     # Update output with the results
     for i, o in enumerate(output):
         # keep the in-source results as definitive
-        if o[4] == "":
```

Comment:
Is the (Scraped) part being handled?

Reply:
Based on the result here it looks like we aren't indicating whether it's an in-source DECOMP_ME_WIP or scraped anymore. I don't mind much either way, just thought I should point this out. https://gist.github.com/Xeeynamo/a41713c9b2a610a67c95199641a19394